dist/partials/query.editor.html

```diff
@@ -1,38 +1,51 @@
-              <p>Select the metric you wish to plot.</p>
-            </info-popover>
-          </div>
-				</div>
-        </div>
-      </div>
+  <div class="gf-form">
+    <label class="gf-form-label width-13 pointer" for="in-query-{{ctrl.target.refId}}">
+      Query
+      <info-popover position="top center" mode="right-normal">
+        <p>Specify a query for the entities you wish to plot. Use the dynamic focus syntax:
+          <a href="https://docs.instana.io/core_concepts/dynamic_focus/#syntax">https://docs.instana.io/core_concepts/dynamic_focus/#syntax</a>
+        </p>
+      </info-popover>
+    </label>
+    <input id="in-query-{{ctrl.target.refId}}" class="gf-form-input" type="text" ng-model='ctrl.target.entityQuery' ng-class="ctrl.target.queryIsValid ? 'ng-valid' : 'ng-invalid'"
+      ng-model-options='{ debounce: 200 }' ng-change="ctrl.onFilterChange(true)" placeholder="Please Specify">
+    </input>
+  </div>
+
+  <gf-form-switch class="gf-form" label="New Application Model"
+    label-class="width-13"
+    checked="ctrl.target.newApplicationModelEnabled"
+    on-change="ctrl.onFilterChange(true)"
+    tooltip="When enabled, the API will return data in the new application model format. Note: Once your Instana environment is fully transitioned to the new application model, this flag will be ignored."></gf-form-switch>
+
+  <div class="gf-form">
+    <label class="gf-form-label width-13 pointer" for="in-type-{{ctrl.target.refId}}">
+      Type
+      <info-popover position="top center" mode="right-normal">
+        <p>Select an entity type for a list of available metrics.</p>
+      </info-popover>
+    </label>
+    <div class="gf-form-select-wrapper width-18 gf-form-select-wrapper--has-help-icon">
+      <select  id="in-type-{{ctrl.target.refId}}" class="gf-form-input ng-pristine ng-valid ng-not-empty ng-touched" ng-model="ctrl.target.entityType" ng-change="ctrl.onEntityTypeSelect(true)"
+        ng-options="f as ctrl.metricsDefinition[f].label for f in ctrl.uniqueEntityTypes">
+        <option value="" disabled selected>{{ctrl.entitySelectionText}}</option>
+      </select>
+    </div>
+  </div>
+
+  <div class="gf-form">
+    <label class="gf-form-label width-13 pointer" for="in-metric-{{ctrl.target.refId}}">
+      Metric
+      <info-popover position="top center" mode="right-normal">
+        <p>Select the metric you wish to plot.</p>
+      </info-popover>
+    </label>
+    <div class="gf-form-select-wrapper width-18 gf-form-select-wrapper--has-help-icon">
+      <select  id="in-metric-{{ctrl.target.refId}}" class="gf-form-input ng-pristine ng-valid ng-not-empty ng-touched" ng-model="ctrl.target.metric" ng-change="ctrl.onMetricSelect()"
+        ng-options="f as f.label + ' (' + f.key + ')' for f in ctrl.availableMetrics">
+        <option value="" disabled selected>{{ctrl.metricSelectionText}}</option>
+      </select>
     </div>
   </div>
-</query-editor-row>
\ No newline at end of file
+</query-editor-row>
```

Comment:
We probably want to make the labels a bit wider as the "New Application Model" is wrapping and pushing the info button down making the label a bit higher:

![image](https://user-images.githubusercontent.com/2222966/41624413-35363db4-73db-11e8-96d4-a3dea2a32e39.png)